bazelisk.go

```diff
@@ -664,12 +663,12 @@ func migrate(bazelPath string, baseArgs []string, newArgs []string) {
 	fmt.Printf("\n\n+++ Result\n\n")
 	fmt.Printf("Command was successful with the following flags:\n")
 	for _, arg := range passList {
-		fmt.Printf("  %s\n", arg)
+		fmt.Printf("  %s (%s)\n", arg, flags[arg])
 	}
 	fmt.Printf("\n")
 	fmt.Printf("Migration is needed for the following flags:\n")
 	for _, arg := range failList {
-		fmt.Printf("  %s\n", arg)
+		fmt.Printf("  %s (%s)\n", arg, flags[arg])
```

Comment:
Can we print “—flag (Bazel 2.0)” here and not just “—flag (2.0)”? I think the latter is hard to understand.

Reply:
Done